service/routes.py

```diff
@@ -111,13 +111,13 @@ def delete_inventory_record(product_id):
     @param: product_id is the id of the record that is to be deleted
     """
     app.logger.info("Request to delete inventory record with id: %s", product_id)
-    inventory = find_from_request_json(request.get_json())
+    inventory = Inventory.find_by_id(product_id)
```

Comment:
since our primary key is id and condition don't we need to use the find function? and pass condition in the url then?

Reply:
your suggestion makes sense, prof suggested to go without condition, we can go ahead with this for now and come back later to pass with condition as well maybe if required. what do you think?

Reply:
resolved with new commit!